core/controllers/get_filters.py

```diff
@@ -66,19 +66,27 @@ def get_outcome_categories():
     return outcome_categories, 200
 
 
-def get_regions():
-    """Returns all unique itl region codes associated with projects.
+def get_geospatial_regions():
+    """Returns all unique ITL1 region codes associated with projects.
 
-    :return: a list of itl region codes
+    :return: a list of ITL1 region codes and region names in dictionaries
     """
-    projects = Project.query.all()
-
-    itl_regions = set(region for project in projects for region in project.itl_regions)
+    geospatial_itl1_regions = (
+        GeospatialDim.query.order_by(GeospatialDim.data_blob["itl1_region_name"])
+        .distinct(GeospatialDim.itl1_region_code, GeospatialDim.data_blob["itl1_region_name"])
+        .with_entities(GeospatialDim.data_blob, GeospatialDim.itl1_region_code)
+        .where(GeospatialDim.projects != None)  # noqa
+        .all()
```

Comment:
Do you mind sharing some context around this data blob? Why do we have `itl1_region_code` as a column but `itl1_region_name` as a key inside a jsonb field? Do we expect to expand this slightly, have different data in the future, etc?

Reply:
It's detailed in a bit more info on the ticket which clarified which should go into the data blob but it's part of moving a lot of the event/non-indexed data to data blobs that Noel and Joel have done over the last few months to make tables more extendable. Here, `itl1_region_code` is a column and indexed because it's what used when someone applies a region filter on Find, but the `itl1_region_name` can go in the data blob as it won't be searched in that way. We can also add more data to the data blob if and when we want more data that matches a postcode prefix.

Reply:
As discussed, `itl1_region_name` will be moved out of the datablob and into its own column as part of a separate ticket (FMD-275 in the backlog)